search/processor.cpp

```diff
@@ -580,8 +668,10 @@ void Processor::InitParams(QueryParams & params)
     params.m_langs.insert(GetLanguage(i));
 }
 
-void Processor::InitGeocoderParams(Geocoder::Params & params, bool viewportSearch)
+void Processor::InitGeocoderParams(Geocoder::Params & params)
```

Comment:
InitGeocoderParams not only inits params, but also inits geocoder. We need to think about new method name.